src/main/java/frc/robot/commands/MoveElevatorCommand.java

```diff
@@ -60,26 +60,31 @@ public MoveElevatorCommand(AutomatedSubsytem.RocketLevel level, boolean cargo)
           encoderTarget = RobotMap.ELEVATOR_LEVEL_3_HATCH_VALUE;
       }
     }
+
+    RobotMap.elevatorSubsystem.setSetpoint(encoderTarget);
     
-    if (RobotMap.elevatorSubsystem.getEncoderPosition() < encoderTarget)
+    /*if (RobotMap.elevatorSubsystem.getEncoderPosition() < encoderTarget)
     {
       goingDown = false;
-    }
+    }*/
   }
 
   /**
    *  Called just before this Command runs the first time
    */
   @Override
-  protected void initialize() {}
+  protected void initialize() 
+  {
+    RobotMap.elevatorSubsystem.enable();
+  }
 
   /**
    * Moves elevator according to whether or not the encoder is higher or lower than the target.
    */
   @Override
   protected void execute() 
   {
-    if (RobotMap.elevatorSubsystem.getEncoderPosition() > encoderTarget)
+    /*if (RobotMap.elevatorSubsystem.getEncoderPosition() > encoderTarget)
```

Comment:
If we're going to comment out the entire execute method wouldn't it make more sense to just delete the command?

Reply:
The command is required for PID

Reply:
using commands provides us access to parallel execution which wouldn't be possible otherwise (at least not as easily)